Flask >= 1.1.2
requests_oauthlib

google-cloud-logging >= 1.15.0
google-cloud-datastore >= 1.12.0
# avoid bug to import google.cloud.datastore - from 1.20.0 to latest 1.21.0 cause AttributeError
google_api_core == 1.16.0
# avoid bug to build grpcio - from 1.27.1 to latest 1.30.0 cause error
grpcio == 1.26.0

flask_gcp_wand >= 0.1.5
